.github/workflows/lint-changed.yml

```
@@ -17,19 +17,41 @@ jobs:
     if: ${{ github.actor != 'OSBotify' || github.event_name == 'workflow_call' }}
     runs-on: ubuntu-latest
     steps:
+      - name: Fetch merge base SHA from API
+        id: merge_base
+        run: |
+          MERGE_BASE_SHA="$(gh api repos/"$REPO"/compare/"$PR_BASE_SHA"..."$PR_HEAD_SHA" \
+          | jq -r '.merge_base_commit.sha')"
+          echo "MERGE_BASE_SHA=$MERGE_BASE_SHA" >> "$GITHUB_OUTPUT"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO: ${{ github.repository	}}
```

Looks like a tab snuck its way in there 😄 

```suggestion
          REPO: ${{ github.repository }}
```

---

I'm so disappointed in myself :sob:.